addons/mail/static/src/models/composer_view/composer_view.js

```diff
@@ -3,13 +3,21 @@
 import { emojis } from '@mail/js/emojis';
 import { registerModel } from '@mail/model/model_core';
 import { attr, many, one } from '@mail/model/model_field';
-import { clear, insertAndReplace, link, replace, unlink, unlinkAll } from '@mail/model/model_field_command';
+import { clear, insert, insertAndReplace, link, replace, unlink, unlinkAll } from '@mail/model/model_field_command';
 import { OnChange } from '@mail/model/model_onchange';
 import { addLink, escapeAndCompactTextContent, parseAndTransform } from '@mail/js/utils';
 import { isEventHandled, markEventHandled } from '@mail/utils/utils';
 
 import { escape } from '@web/core/utils/strings';
 
+const getMessageCompositionId = (function() {
+    let tmpId = 0;
+    return () => {
+        tmpId -= 1;
+        return tmpId;
+    };
+})();
```

Comment:
We did this a lot, but it's not great to have floating functions that are not owned by a model.
And also it's stateful without being in the modelling.

For instance, this could be a static method of the model at hand, and the next id could be stored in global object like `Messaging`.

We gonna clean all of them, so maybe it's fine for now.
I don't think we need this function for `MessageComposition/id` anyway.

Reply:
There is a clear need of cleaning those wild functions. That is out of the scope of this PR of course.